homeassistant/components/zwave_js/config_flow.py

```diff
@@ -134,9 +134,7 @@ async def async_step_manual(
             step_id="manual", data_schema=STEP_USER_DATA_SCHEMA, errors=errors
         )
 
-    async def async_step_hassio(  # type: ignore # override
-        self, discovery_info: dict[str, Any]
-    ) -> dict[str, Any]:
+    async def async_step_hassio(self, discovery_info: dict[str, Any]) -> FlowResultDict:  # type: ignore
```

Comment:
Can we use the more specific `# type: ignore[override]`?

https://mypy.readthedocs.io/en/stable/common_issues.html#incompatible-overrides

Reply:
Sure! I tried to fix it in this PR but it requires more places to fix. Will probably address this in another PR.